### DISCUSSION TITLE
Running AquaCrop standalone on any platform

### DISCUSSION THREAD

**Discussion by gdelannoy** [KUL-RSDA/AquaCrop#323] [Show and tell]
If you have an existing working environment for earlier releases of AquaCrop, or if you want to try out the testcase with a new executable on any platform, then:

1. copy the executable to the present directory, next to the folders with INPUT, LIST, OUTP, PARAM, etc., and
2. follow the instructions below for your operating system (pay attention to the direction of the "/" and "\\"):

* Linux:

  Open a terminal, change to the present directory and execute
  "./aquacrop".

  If it fails, then [check this discussion post.](https://github.com/KUL-RSDA/AquaCrop/discussions/318#discussion-4459645) 

* MacOS:

  Open a Terminal, change to the present directory ($cd [your_path]) and
  execute "./aquacrop". If you receive a security alert upon runtime, then locate
  the AquaCrop executable in your Finder, CTRL+click on the executable
  and click on "Open". Then, execute "./aquacrop".

* Windows:

  You can either:
  - Open a Windows Powershell or CMD shell, change to the present
    directory and execute ".\aquacrop.exe",
  - Or double-click on the executable in the present directory.
    An empty CMD window may appear while AquaCrop is running.

[This is copy-paste of the README.md in the testcase zip-file]

**Comment by WAMBUA12** [comment]
I have encountered this problem while trying to run Aqua crop standalone in windows where the project runs but the output doesn't have any data if the calendar dates are between 1st January and 1st March. However, it runs well for any other date outside these dates. What could be the problem?

**Reply by mbechtold** [reply]
Could you please set up the same simulation in the AquaCrop GUI. Does it give you output there? If yes, save it as a project and apply again the windows standalone on the saved project. If the standalone does give you something else, please provide all project files to us that we can reproduce the issue. Thanks.

**Reply by WAMBUA12** [reply]
Yes, it does run well in the AquaCrop GUI. However, it doesn't write any results in the standalone version of AquaCrop. I am using AquaCrop version. 7.0.

**Comment by WAMBUA12** [comment]
[My_project_Files.zip](https://github.com/KUL-RSDA/AquaCrop/files/13398833/My_project_Files.zip)
Kindly find attached the project files as earlier requested. I hope I have zipped all the necessary files. For CO2 I used the MAunaloa.co2. Looking forward to your feedback.

**Reply by lbusschaert** [reply]
[Bura_Rice2_HistoricalPRMday.txt](https://github.com/KUL-RSDA/AquaCrop/files/13425051/Bura_Rice2_HistoricalPRMday.txt)
@WAMBUA12 Attached, you can find the output generated from your test case with the Windows AquaCrop standalone (V7.0). The daily output file is called _Bura_Rice2_HistoricalPRMday.OUT_. It seems like AquaCrop is running correctly for the calendar days between the dates you mentioned. 
If you were looking at the seasonal output file (_Bura_Rice2_Historicalseason.OUT_), the dates are referring to the end of the simulation period, and the totals (cumulated values) are written in the output.

If you don't get the daily output file _Bura_Rice2_HistoricalPRMday.OUT_, you may have forgotten to specify that you wanted daily output. This can be done by adding a file called _DailyResults.SIM_ in the SIMUL folder, and add the following lines in this file:

>  1 : Various parameters of the soil water balance
>  2 : Crop development and production
>  3 : Soil water content in the soil profile and root zone
>  4 : Soil salinity in the soil profile and root zone
>  5 : Soil water content at various depths of the soil profile
>  6 : Soil salinity at various depths of the soil profile
>  7 : Climate input parameters

I hope this helps. If you would still encounter any issues, do not hesitate to reply in this thread.

**Comment by WAMBUA12** [comment]
Dear Dr/Mr,

Thank you for your assistant. I will get back to you soon in case of any
problem.
Regards,
